openmc/data/njoy.py

```diff
@@ -573,21 +580,18 @@ def make_ace_thermal(filename, filename_thermal, temperatures=None,
         commands += _THERMAL_TEMPLATE_ACER.format(**locals())
 
         # Indicate tapes to save for each ACER run
-        tapeout[nace] = fname.format(ace, temperature)
-        tapeout[ndir] = fname.format(xsdir, temperature)
+        tapeout[nace] = output_dir / fname.format(ace, temperature)
+        tapeout[ndir] = output_dir / fname.format(xsdir, temperature)
```

Comment:
Should this be `"xsdir"` now that the default value of `xsdir` is `None`?

Reply:
Yes, you're absolutely right. Will get this fixed; thanks!